src/dmd/expressionsem.d

```diff
@@ -1885,27 +1885,31 @@ private bool functionParameters(const ref Loc loc, Scope* sc,
              *   S().bar() = 1;      // bad!
              * }
              */
-            Dsymbol s = null;
-            if (fd.isThis() || fd.isNested())
-                s = fd.toParent2();
-            for (; s; s = s.toParent2())
+            bool checkEclosingWild(Dsymbol s)
```

Comment:
checkEclosing --> checkEnclosing  Missing an 'n'.

Reply:
If we could teach the compiler English grammar. Thanks!